docs/cloud/pouta/vm-flavors-and-billing.md

```diff
@@ -205,6 +209,15 @@ flavors have faster CPUs and no overcommitment of CPU cores.
 
 **cPouta HPC flavor characteristics:**
 
+**hpc.5.\*:**
+
+-   No redundant power
```

Comment:
There is redundant power. Please update

- this text statement
- the circle icons in the table (I think "p0" needs to change to "p100" or something like that)

Reply:
This issues have been fixed by a new commit.